### DISCUSSION TITLE
Windows Advertising SDK Support?

### DISCUSSION THREAD

**Discussion by vector-man** [AvaloniaUI/Avalonia#7681] [Q&A] [answered]
I'm looking into adding advertising to an upcoming Windows application. There's a number of ad SDKs for Windows. Is there any any way I can use them in Avalonia? (I'm specifically looking at Microsoft's and AdJumbo's right now.)

**Comment by maxkatz6** [comment, marked as answer]
Do these SDKs work with .NET Core? Do these SDK needs platform specific implementation (if yes, do they have it for macos, linux and possibly more?). 
If "yes" is the answer to all of these questions, it can be integrated with avalonia. Either directly rendering or through NativeControlHost.

**Reply by maxkatz6** [reply]
For instance, AdsJumbo has winforms SDK, which means you can embed it using NativeControlHost, should work. But only on Windows. Unless they have implementation of this SDK for macos cocoa somewhere, and for linux.
